packages/app/src/components/Survey/SurveyPresentational.tsx

```diff
@@ -99,15 +100,49 @@ export const SurveyPresentational: React.FC<SurveyPresentationalProps> = ({
 
                 <Box>
                     <Heading use="h3" marginBottom="major-3">
-                        {intl.formatMessage({
-                            id: "statistics",
-                            defaultMessage: "Statistics"
-                        })}
+                        Statistics
                     </Heading>
-
-                    TODO
+                    <Stack spacing="major-2">
+                        <Heading use="h4">
+                            eNPS
+                        </Heading>
+                        <Table>
```

Comment:
When there is no enps survey, this view should not show. i.e 
<Table if(enps)> ...  </Table>